### ISSUE TITLE
To do: Prezi

### ISSUE THREAD

**Issue by EstherDeLoof** [OpenWebslides/ICT-Lerarenopleiding#6]
The prezi presentation is very slow when embedded. Alternatively we could add the jpg image of the prezi and scroll through it like we did in our pitch with the timeline. Can this be done?
The PreziImage had already been added to the images folder.

**Comment by RubenVerborgh** [comment]
Yeah… Prezi is still written in Adobe Flash, which is [a horrible artefact from the past that has very few to do with the Web](http://www.apple.com/hotnews/thoughts-on-flash/).

On my machine, the speed is quite okay, but this is likely platform-specific.

**Comment by EstherDeLoof** [comment]
OK, good to know that prezi has an achilles heel so we can avoid it in the future.

**Comment by EstherDeLoof** [comment]
I've been trying to add a scroll to the backup of the prezi image but it doesn't work. It had been commented out but I'm very curious why it didn't work.

**Comment by RubenVerborgh** [comment]
It's a bit more complicated than that. The surrounding element needs to have fixed width and height, otherwise it will just fit to the contents (so nothing to overflow).
